### ISSUE TITLE
Allow manual override for updating & publishing docs

### ISSUE THREAD

**Issue by stephenplusplus** [GoogleCloudPlatform/google-cloud-node#1618]
Currently, CI:

  1. runs after a merge to master
  1. notices a change in a method's docs
  1. updates the docs site

There is a bit of a problem if the build fails (flakiness) or if we cancel it. The docs would end up not being built until the next time there's a merge and its tests pass. It would be nice to have an override so that we don't have to a) wait on CI if we don't feel like it and b) update the docs ourselves in the event something goes wrong with the build.
